### ISSUE TITLE
Optionally include Copyrights/Notices in scan results

### ISSUE THREAD

**Issue by anders-swanson** [aquasecurity/trivy#1639]
Is it possible for Trivy to include Copyrights and/or Notice texts in scan results, if they are present? This could be a helpful compliance feature.

**Comment by omkarparkhe** [comment]
I agree with @anders-swanson. It would be very help in terms license compliance and also benefits the user/developer to readily consume a notices file with all the relevant licenses and copyright data from the SBOM generated by Trivy.
